### ISSUE TITLE
Hello I fixed a broken  version error in the setup.py.

### ISSUE THREAD

**Issue by SFMG2001** [KrispyCamel4u/SysMonTask#133]
Hello,
I was installing sysmontask from source code and noticed when I ran it it refused to run with the error:

'version 1.x.x is not valid'

also told me to look at python's PEP440 or whatever. Decided nah I'm curious so I went in and found what it was complaining about. apparently when running setup.py it looks for the version string and when it comes across '1.x.x' it gets confused then fails. setting that to a valid version number fixed it. Hope this helps! :D

Kevin M.
SFMG2001

Manjaro Linux

**Comment by RedSnt** [comment]
I'm getting the same errors. According to a quick internet search, results claim that it's due to an invalid version scheme. If you check setup.py with a text editor, on line 18 there's ```version='1.x.x',``` - changing that to, say 1.0.0 helped me compile it correctly.  
(On a 2009, 1GB ram, 32-bit (1 core) 2 thread CPU EeePC running Q4OS (Debian)).  
It works just fine 👍

**Comment by SFMG2001** [comment]
> I'm getting the same errors. According to a quick internet search, results claim that it's due to an invalid version scheme. If you check setup.py with a text editor, on line 18 there's `version='1.x.x',` - changing that to, say 1.0.0 helped me compile it correctly. (On a 2009, 1GB ram, 32-bit (1 core) 2 thread CPU EeePC running Q4OS (Debian)). It works just fine 👍

indeed! I wonder if he'll implement the fix or is this project dead?

**Comment by Danny3** [comment]
This project is dead!

Everyone should move to the actively developed and maintained Mission Center:
https://missioncenter.io/
https://flathub.org/en/apps/io.missioncenter.MissionCenter

For GPU information and statistics only, there is also GPU-T:
https://github.com/lseurttyuu/GPU-T

And for AMD users only, there is also this great information and statistics GUI + TUI called AMDGPU_TOP:
https://github.com/Umio-Yasuno/amdgpu_top
